server/core/include/scoped_permission.hpp

```diff
@@ -78,6 +81,8 @@ namespace irods::experimental
         RsComm& comm_;
         const filesystem::path path_;
         filesystem::perms old_perm_;
+        /// \var Automatically constructed username composed of the username and zone name, joined by a pound sign.
```

Comment:
I don't think we need to document this. It is pretty self explanatory IMO.

Reply:
Done